src/connection/NeodashRecordParser.ts

```diff
@@ -0,0 +1,27 @@
+import { NeodashRecord } from './NeodashRecord';
+
+const notImplementedError = (functionName: string): never => {
+  throw new Error(`Not Implemented: ${functionName}`);
+};
+
+export abstract class NeodashRecordParser {
```

Comment:
this will break right? `bulkParse` for tables?

Maybe we can just leave out the record parser in this iteration of the code.